sdk/Sdk/Targets/Microsoft.Azure.Functions.Worker.Sdk.Publish.targets

```diff
@@ -0,0 +1,78 @@
+Microsoft.Azure.Functions.Worker.Sdk.Publish.targets
+
+WARNING:  DO NOT MODIFY this file unless you are knowledgeable about MSBuild and have
+          created a backup copy.  Incorrect changes to this file will make it
+          impossible to load or build your projects from the command-line or the IDE.
+
+***********************************************************************************************
+-->
+
+<Project ToolsVersion="14.0"
+         xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+
+  <Import Sdk="Microsoft.NET.Sdk.Publish"
+          Project="Sdk.targets" />
+
+  <Import Project="$(MSBuildThisFileDirectory)Microsoft.NET.Sdk.Functions.Publish.$(PublishProtocol).targets"  
+          Condition="Exists('$(MSBuildThisFileDirectory)Microsoft.NET.Sdk.Functions.Publish.$(PublishProtocol).targets')" />
+
+  <Import Project="$(MSBuildThisFileDirectory)Microsoft.Azure.Functions.Worker.Sdk.Publish.$(PublishProtocol).targets"  
+          Condition="Exists('$(MSBuildThisFileDirectory)Microsoft.Azure.Functions.Worker.Sdk.Publish.$(PublishProtocol).targets')" />
+
+  <!--
+    ============================================================
+                  _InitializeDeployOnBuildProperties
+
+    Initializes the publish properties.
+    ============================================================
+    -->
+
+  <Target Name="_InitializeDeployOnBuildProperties" >
+
+    <ConvertToAbsolutePath Paths="$(PublishIntermediateOutputPath)">
+      <Output TaskParameter="AbsolutePaths"
+              PropertyName="PublishIntermediateOutputPath"/>
+    </ConvertToAbsolutePath>
+
+    <PropertyGroup>
+      <PublishDir>$(PublishIntermediateOutputPath)</PublishDir>
+      <PublishDir Condition="!HasTrailingSlash('$(PublishDir)')">$(PublishDir)\</PublishDir>
+    </PropertyGroup>
+
+    <!-- Remove all the files from the temp directory first-->
+    <ItemGroup>
+      <_PublishTempFiles Include="$(PublishIntermediateOutputPath)**\*.*" />
+    </ItemGroup>
+
+    <Delete Files="@(_PublishTempFiles)"
+            ContinueOnError="true" />
+
+    <RemoveDir Directories="$(PublishIntermediateOutputPath)"
+               ContinueOnError="true"
+               Condition="Exists('$(PublishIntermediateOutputPath)')" />
+
+    <MakeDir Directories="$(PublishIntermediateOutputPath)"
+             Condition="!Exists('$(PublishIntermediateOutputPath)')" />
+
+  </Target>
+
+  <!--
+    ============================================================
+
+    This targets gets called when publish is invoked with DeployOnBuild
+    set. This target is responsible for overriding the publish targets
+    from Publish SDK and generating function.json during publish.
+
+    ============================================================
+    -->
+  <PropertyGroup>
+    <CorePublishDependsOn>
+      _InitializeDeployOnBuildProperties;
+      Publish;
+      $(_DotNetPublishFiles);
+    </CorePublishDependsOn>
+  </PropertyGroup>
```

Comment:
Actually I think this block and the tasks referenced are already there in "Microsoft.Azure.Functions.Worker.Sdk.targets" so I will remove these.